nrt/data/IEEE14/IEEE14_SyntaxExamples/IEEE14_BlackBoxModels/reference/outputs/initValues/globalInit/dumpInitValues-GEN____3_SM.txt

```diff
@@ -1,53 +1,53 @@
  ====== INIT VARIABLES VALUES ======
-generator_lambdaDPu                               : y =       1.015355 yp =       0.000024
-generator_lambdaQ1Pu                              : y =      -0.000002 yp =       0.000034
-generator_lambdaQ2Pu                              : y =      -0.000002 yp =       0.000185
-generator_lambdafPu                               : y =       1.131221 yp =      -0.000005
-generator_omegaPu_value                           : y =       1.000000 yp =       0.000005
+generator_lambdaDPu                               : y =       1.015355 yp =      -0.000019
```

Comment:
value not the same as before, why ?

Reply:
Right, due to a few changes in the parameters values (to harmonize things between the different parts of the simulation).
The most important changes are on tolerances for init (local and global) and in particular on the msbset for local initialization (J update was done at each iteration, which is not really necessary). Applying these parameters with the current implementation lead to similar initialisation values than before - I have double checked it.